jmclient/test/test_wallet.py

```diff
@@ -93,6 +93,9 @@ def test_bip49_seed(monkeypatch, setup_wallet):
     assert (mnemonic, None) == wallet.get_mnemonic_words()
     assert account0_xpriv == wallet.get_bip32_priv_export(0)
     assert addr0_script_hash == hexlify(wallet.get_external_script(0)[2:-1])
+
+    # FIXME: is this desired behaviour? BIP49 wallet will not return xpriv for
```

Comment:
Looking at this I actually think a user would want to be able to export the real root key, not just the key derived for the BIP49 base path. Any thoughts?